src/ceph-volume/ceph_volume/util/disk.py

```diff
@@ -320,6 +320,8 @@ def lsblk_all(device='', columns=None, abspath=False):
         base_command.append('-p')
     base_command.append('-o')
     base_command.append(','.join(columns))
+    if device:
+        base_command.append(device)
```

Comment:
You probably should add `--nodeps` in case the device has partitions or so.

```suggestion
        base_command.append('--nodeps')
        base_command.append(device)
```

Reply:
I remember I removed it as part of a refactor for a specific reason but I can't remember why exactly 😳 
I would have to drill down a bit to remember 😅